app/cli/cmd/integration_list.go

```diff
@@ -50,13 +50,13 @@ func integrationListTableOutput(items []*action.IntegrationItem) error {
 	}
 
 	t := newTableWriter()
-	t.AppendHeader(table.Row{"ID", "Name", "Config", "Created At"})
+	t.AppendHeader(table.Row{"ID", "Display Name", "Kind", "Config", "Created At"})
```

Comment:
I renamed `name` for `kind` for consistency. I changed it to `name` in a previous patch but I think it was a mistake, we need to think it through.